### PR TITLE
feat: expose apis of resourcemanager folders

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+          new Function<
+              ConstraintInfo, com.google.api.services.cloudresourcemanager.model.Constraint>() {
+            @Override
+            public com.google.api.services.cloudresourcemanager.model.Constraint apply(
+                ConstraintInfo constraintInfo) {
+              return constraintInfo.toPb();
+            }
+          };
+
+  private static final long serialVersionUID = 9148970963697734236L;
```
Comment (rahulKQL) [inline src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java]: Move it to line 27, just after the class parenthesis starts. Applies to below classes

File: src/main/java/com/google/cloud/resourcemanager/OrgPolicy.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  public final boolean equals(Object obj) {
+    if (obj == this) {
+      return true;
+    }
+    if (obj == null || !obj.getClass().equals(OrgPolicy.class)) {
+      return false;
+    }
+    OrgPolicy other = (OrgPolicy) obj;
+    return Objects.equals(toPb(), other.toPb()) && Objects.equals(options, other.options);
+  }
```
Comment (rahulKQL) [inline src/main/java/com/google/cloud/resourcemanager/OrgPolicy.java:122]: Please be consistent, The implementation is different in previous and next classes.

**Review by rahulKQL — APPROVED** [review]
just minor nits, rest LGTM

File: src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java (hunk 1 above)
Comment (athakor) [reply src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java]: done

**Pull request closed without merging**